### DISCUSSION TITLE
Am doing IOS Automatic Code Signing, but am getting a "Did Not Find Matching Provisioning Profiles for Code Signing" Error when trying to build

### DISCUSSION THREAD

**Discussion by A7f7o7** [codemagic-ci-cd/codemagic-docs#1258] [Q&A] [answered]
I am trying to distribute a Flutter App to IOS App Store via Automatic Code Signing, and have tried selecting Ad Hoc, Development, and App Store but am still getting the "Did Not Find Matching Provisioning Profile" error. Isn't Automatic Code Signing supposed to make the Provisioning Profile itself anyways? 

I have the API Key generated and integrated to CodeMagic, I made a Bundle Identifier and select it in the Workflow Editor. I don't know what the issue is. I'm not familiar with publishing apps to the app store, so I tried to do it in the most simple way possible. I'm not sure if I'm missing something, but I don't know where to go from here.

**Comment by icarusdust** [comment, marked as answer]
Hi @A7f7o7 , to begin with double check your project bundle id and App Store Connect project bundle id (a bundle id that you choose from the dropdown in the Workflow Editor). Sharing a build id with me here would be helpful as well.

**Reply by A7f7o7** [reply]
Hello, I am just seeing this but I actually ended up changing the bundle id and it fixed my problem. So, your suggestion was spot on, thank you
